doc/user_guide.md

```diff
@@ -0,0 +1,324 @@
+
+Pick a (unique) function app name.
+
+Pick '.NET' as the runtime stack and pick '3.1' as the version.
+
+Pick the region that fits you the most (this will probably the same region as where you host the Azure Data Factory as well).
+
+![](./img/2021-08-04-15-08-55-image.png)
+
+Next, press 'Hosting'. This will bring you to the next setup screen.
+
+![](./img/2021-08-04-15-11-27-image.png)
+
+Create a new storage account name (recommended) or use an existing one. 
+
+Take note of the name.
+
+As operating system pick 'Windows', ()this is because we use a .dll from the Exasol ADO .NET data provider that's not crossplatform.)
+
+Press 'Review + Create' and then press ''Create'.
+
+You'll get a notification saying 'Deployment is underway'. This might take a minute.
+
+Once the deployment is complete you'll get a notification and you can navigate to the resource. Do so.
+
+![](./img/2021-08-04-15-14-28-image.png)
+
+We've now succesfully created our azure functions app. You can already take a look around. 
+
+As you see there are no functions yet so our next step will be uploading these functions.
+
+For this we'll use the Azure CLI tools.
+
+#### Install the Azure CLI (get it here):
+
+[Install the Azure CLI for Windows | Microsoft Docs](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli-windows?tabs=azure-cli)
+
+#### Get the zipped release of the Exasol azure helper functions
+
+Get the packaged version of the exasol azure data factory functions here:
+
+Release.zip (TODO: add url)
+
+#### Deploy the zipped functions using the azure CLI
+
+Using the provided .zip in the release you can easily deploy the azure functions.
+
+Download the .zip
+
+Use azure cli:
+
+Login:
+
+Open a command line window ("cmd") or powershell or a terminal of your choice
+
+Type:
+
+az login
+
+Your preferred browser will open and you will be able to login.
+
+Now navigate to the folder where you downloaded the packaged zip file and run the following command:
+
+az webapp deployment source config-zip -g $resourceGroupName -n $functionName --src $sourceZipPath
+
+In our case this would be
+
+az webapp deployment source config-zip -g adf-ug-rg -n exaadfug --src adffunctions.zip .
+
+Running this command might take a while.
+
+If everything is succesfull you'll get a JSON response describing the resource and that its ProvisioningState "succeeded".
+
+![](./img/2021-08-04-15-22-00-image.png)
+
+If you navigate back to your Function App and click on the Functions tab you will see that the helper functions are now present:
+
+<img src="file:///C:/adftut/doc/img/2021-08-04-15-28-19-image.png" title="" alt="" width="747">
+
+### Alternate approach : Deploying to Azure using Visual Studio or Visual Code ''Publish''
+
+These methods may be a bit more involved and require you to have visual studio or visual studio code installed but might be more practical if you want to alter or add some of the behaviour of these functions yourself.
```

Comment:
```suggestion
These methods may be a bit more involved and require you to have a visual studio or visual studio code installed but might be more practical if you want to alter or add some of the behavior of these functions yourself.
```

Reply:
(both are correct behavior and behaviour)